src/LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Would you please explain what is the rationale behind choosing GPL V3?

Reply:
The rayTracing module was replaced by a custom rayTracing module originated from a literature reference and not from a library, i.e. no GPL V3 reference is necessary anymore